index.html

```diff
@@ -420,71 +422,156 @@ <h4>Creating a new Protocol Binding Template Subspecification</h4>
+                </p>
+                <section id="payload-bindings-contentType">
+                    <h5>Content Types</h5>
+
+                    <p>
+                        Content type includes the media type and potential parameters for the media type and it enables
+                         proper processing of the serialized documents.
+                        This way, the messages can be exchanged in any format and allow the upper layers of an application
+                        to adapt to different formats.
+                        In some cases such as images, videos or any unstructured data, content type is enough to describe the
+                        payload but in cases like JSON ([[RFC8259]]) a Data Schema is usually provided, like explained in [[[#payload-bindings-dataschema]]].
+                    </p>
+                    <p>
+                        For example, a number payload can be serialized as JSON or XML and be indicated in the <code>contentType</code>
+                        of the forms with <code>application/json</code> or <code>application/xml</code>, respectively.
+                        Further parametrization is possible via the plus (<code>+</code>) or the semicolon (<code>;</code>)
+                        notations.
+                    </p>
+
+                    <p>
+                        In the example below, you can find the form elements with content types for JSON and text payloads 
+                        such as JSON and CBOR with additional parameters.
+                        In this specific case, the forms describe that reading this property with <code>http</code> or 
+                        <code>coap</code> result in different content types.
+                        For structured media types, a Data Schema is generally provided in the affordance level as 
+                        explained in [[[#payload-bindings-dataschema]]] and <a href="https://w3c.github.io/wot-thing-description/#sec-data-schema-vocabulary-definition">
+                        in the Data Schema section of the TD specification</a>.
+                        However, for unstructured data such as images and videos, a Data Schema is typically not available.
+                    </p>
+
+                    <pre class="example" title="JSON and CBOR media types in forms" id="example-payload-binding">
+                        {
+                            "forms":[
+                            {
+                                "href": "http://example.com/properties/temperature",
+                                "op": "readproperty",
+                                "contentType": "application/json"
+                            },
+                            {
+                                "href": "coap://example.com/properties/temperature",
+                                "op": "readproperty",
+                                "contentType": "application/cbor;charset=utf-8"
```

Comment:
I am not a CBOR expert but i was surprised to read 
`contentType": "application/cbor;charset=utf-8`
since CBOR is binary-encoded. Isn't that the case?
Maybe `charset` is better suited for `text/plain;charset="UTF-8`

Reply:
I have fixed this